packages/cubejs-query-orchestrator/src/driver/BaseDriver.ts

```diff
@@ -86,17 +87,14 @@ const DbTypeValueMatcher = {
   text: () => true
 };
 
-export class BaseDriver {
-  /**
-   * Workaround for Type 'BaseDriver' has no construct signatures.
-   *
-   * @param {Object} [options]
-   */
-  constructor(_options = {}) {
+export class BaseDriver implements DriverInterface {
```

Comment:
if we are going to migrate this class to TypeScript, maybe it's a good idea to make `abstract`?